examples/failure-notification/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+## v0.0.1 (2022.09.30)
+
+### failure notifications
+
+This Workflow Template is an example of parsing and sending a slack notification on failure of a worklow.
```

Comment:
workflow (last one) is misspelled online #7

Reply:
Fixed misspelling